core/src/Platforms/iOS/iOSPlatformProxy.cs

```diff
@@ -120,6 +122,20 @@ public string GetProductName()
         {
             return _isMsal ? "MSAL.Xamarin.iOS" : "PCL.iOS";
         }
+        public string GetCallingApplicationName()
+        {
+            return (NSString)NSBundle.MainBundle?.InfoDictionary?["CFBundleName"];
```

Comment:
If these are empty, how will that impact our telemetry data?  Will we get it keyed off of "null" or will it be dropped?  If it will be dropped if null, i'd suggest having a default value for these like "unknown_calling_application" and "unknown_device_id" so taht we can bucketize the counts of telemetry that are failing with unknown/uncalculatable values.

Reply:
In case of these values being null, we would still get a telemetry batch with these fields - they would just be empty.